build.py

```diff
@@ -192,6 +192,8 @@ def validate_config(model_path: str):
             " directory (or hf-path) that contains the pre-compiled model in raw HuggingFace"
             " format instead.".format(model_path)
         )
+    if model_path.split("/")[-1].startswith("minigpt"):
```

Comment:
oh yeah, it's because when building minigpt, unlike other models we specify a model folder containing "config.json", in minigpt I would just let users specify a single model file containing the weights. Thus I'd like to skip validating "config.json" for minigpt because minigpt does not have a config.json file by default.

The authors of minigpt only provides a single weight file for download and not any config.json file: https://drive.google.com/file/d/1RY9jV0dyqLX-o38LrumkKRh6Jtaop58R/view?usp=sharing

Reply:
I added a comment for the line in the code. Let me know if you have any further questions!

Reply:
Thank you for your explaination!